ngraph/test/files/paddlepaddle/gen_scripts/generate_interpolate.py

```diff
@@ -32,7 +32,7 @@ def pdpd_interpolate(x, sizes=None, scale_factor=None, mode='nearest', align_cor
         interp = interpolate(node_x, size=sizes, scale_factor=scale_factor,
                              mode=mode, align_corners=align_corners, align_mode=align_mode,
                              data_format=data_format, name=name)
-        out = pdpd.static.nn.batch_norm(interp, use_global_stats=True, epsilon=0)
```

Comment:
This change is fine but it is not necessary, it is an issue recently introduced with new changes in nGraph (PR https://github.com/openvinotoolkit/openvino/pull/5468). I've created issue #55721 on other team for this

Reply:
Got it, for now I just remove this to let the unit test pass.